addons/medical_treatment/ACE_Medical_Treatment_Actions.hpp

```diff
@@ -239,20 +239,30 @@ class GVAR(actions) {
 
     // - Misc -----------------------------------------------------------------
     class BodyBag: BasicBandage {
-        displayName = CSTRING(PlaceInBodyBag);
+        displayName = CSTRING(PlaceInBodyBagBlack);
         displayNameProgress = CSTRING(PlacingInBodyBag);
         icon = QPATHTOEF(medical_gui,ui\bodybag.paa);
         category = "advanced";
         treatmentLocations = TREATMENT_LOCATIONS_ALL;
         allowSelfTreatment = 0;
         medicRequired = 0;
         treatmentTime = QGVAR(treatmentTimeBodyBag);
-        items[] = {"ACE_bodyBag", "ACE_bodyBag_blue", "ACE_bodyBag_white"};
+        items[] = {"ACE_bodyBag"};
         condition = QFUNC(canPlaceInBodyBag);
         callbackSuccess = QFUNC(placeInBodyBag);
         consumeItem = 1;
         litter[] = {};
     };
+    class BodyBagBlue: BodyBag {
+        displayName = CSTRING(PlaceInBodyBagBlue);
+        icon = QPATHTOEF(medical_gui,ui\bodybag.paa);
```

Comment:
Uh, does this need to be redefined?

Reply:
It is already just a white icon for the interaction menu, can we define the color here rather than add two more images?

Reply:
I think we need to add support for it but sure.

Reply:
I meant does the property need to be redefined though, it's the same value for the three of them.

Reply:
that's fine right? it's just the interaction icon

Reply:
it's fine, could also just yeet it /shrug

Reply:
oh lol, I just realized what you're saying. yeah that was me not checking github copilot's work close enough.,..